features-json/css-writing-mode.json

```diff
@@ -0,0 +1,232 @@
+{
+  "title":"CSS Writing Mode",
+  "description":"Property to define whether lines of text are laid out horizontally or vertically and the direction in which blocks progress.",
+  "spec":"https://drafts.csswg.org/css-writing-modes-3/#block-flow",
+  "status":"cr",
+  "links":[
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode",
+      "title":"MDN article"
+    }
+  ],
+  "bugs":[
+
+  ],
+  "categories":[
+    "CSS"
+  ],
+  "stats":{
+    "ie":{
+      "5.5":"n",
```

Comment:
This has been supported in IE since 5.5:
https://msdn.microsoft.com/en-us/library/bb250415(v=vs.85).aspx
http://generatedcontent.org/post/45384206019/writing-modes